ops/Dockerfile

```diff
@@ -183,6 +183,18 @@ ENV CCACHE_DIR=/ccache
 
 RUN --mount=type=cache,target=/ccache/ source /conda/dev/bin/activate && ninja install
 
+ARG CCACHE_REMOTE_STORAGE
+RUN if [ -n "${CCACHE_REMOTE_STORAGE}" ] ; then \
+      apt-get update -qq && apt-get install -y ccache ; \
```

Comment:
Why are we installing ccache here? It is already available in the conda build environment?

Reply:
I can drop this installation.